.github/workflows/snaefell-ui.yml

```diff
@@ -2,6 +2,8 @@ name: Snaefell UI CI/CD
 
 on:
   push:
+    branches-ignore:
```

Comment:
do we need this, as dependabot will not be able to push to main using `snaefull-ui-v` tag.

Reply:
@dantaik I'm not sure if it's working as intended because you can see [here](https://github.com/taikoxyz/taiko-mono/actions/runs/10822019392/job/30025140228?pr=18074) that dependabot PRs are triggering the workflows even without the `snaefell-ui-v` tag.

Reply:
Then maybe we should disable all vercel actions for the bot in repo--vercel-deploy.yaml

Reply:
I made the above change in a upstream PR.